docs/preprocessor/gen-options-md.py

```
@@ -0,0 +1,84 @@
+"""
+
+import json
+import sys
+
+
+def render_value(val):
+    """Collapse a NixOS literalExpression / literalMD / plain value to a string."""
+    if val is None:
+        return None
+    if isinstance(val, dict):
+        type = val.get("_type", "")
```

Ruff (0.15.11)

[error] Variable `type` is shadowing a Python builtin

(A001)